lib/ecto/enum.ex

```diff
@@ -81,6 +81,7 @@ defmodule Ecto.Enum do
   """
 
   use Ecto.ParameterizedType
+  import Kernel, except: [inspect: 1]
```

Comment:
yeah let's definitely call the callback something else. How about `format_error` until we come up with something better?

Reply:
I think `inspect` is fine. It is exact the purpose we expect it to have.

Reply:
Nah, I see that it will be confusing with each one we are calling. Let's rename it to `format`.